tasks/terraform_apply.json

```diff
@@ -0,0 +1,27 @@
+{
+  "description": "Apply an HCL manifest",
+  "files": ["ruby_task_helper/files/task_helper.rb", "terraform/lib/cli_helper.rb"],
+  "input_method": "stdin",
+  "parameters": {
+    "dir": {
+      "type": "Optional[String[1]]",
+      "description": "Path to Terraform project directory. Path is relative to Boltdir, unless an absolute path is specified."
```

Comment:
Since I don't think this configuration will typically come from a config file it seems like it should be relative to the cwd. I think we do it relative to the boltdir for plugins since that config most likely comes from the inventory or config.